tests/src/whisk/core/controller/test/PackagesApiTests.scala

```diff
@@ -401,48 +400,7 @@ class PackagesApiTests extends ControllerTestCommon with WhiskPackagesApi {
         put(entityStore, reference)
         Put(s"$collectionPath/$aname", content) ~> sealRoute(routes(creds)) ~> check {
             status should be(BadRequest)
-            responseAs[ErrorResponse].error should include("cannot bind to another package binding")
-        }
-    }
-
```

Comment:
since whisk entities apply to triggers and rules, the test coverage should extend to those apis as well.